### PR TITLE
Item-Descriptions

### PR DESCRIPTION
**Pull request by armorheart**
Adds bunch of detailed descriptions to some weapons and shields. Will later add more descriptions to armors, clothing and other items with zero description.

### CODE REVIEW

File: code/game/objects/items/rogueweapons/melee/swords.dm [hunk 1]
```diff
@@ -88,7 +88,7 @@
 	lefthand_file = 'icons/mob/inhands/weapons/roguebig_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/roguebig_righthand.dmi'
 	name = "bastard sword"
-	desc = ""
+	desc = "A serious weapon to behol. Large, razor sharp, and undeniably lethal. Offers very high damage output and even more when wielded. This weapon is gods right hand."
```
Comment (CVSthePharmacy) [inline code/game/objects/items/rogueweapons/melee/swords.dm:91]: there's a typo here

**Comment by CVSthePharmacy** [comment]
also why do none of the sabres have their damage logged, nor do the rapiers nor the iron swords

File: code/game/objects/items/rogueweapons/melee/swords.dm (hunk 1 above)
Comment (armorheart) [reply code/game/objects/items/rogueweapons/melee/swords.dm:91]: Fixing in hotfix. thanks

**Comment by armorheart** [comment]
> (Just some tweaks)

Thanks! Our english is different from the british or american english so these tweaks are greatly welcome to suit the wider audience.

**Comment by ChungusGamer666** [comment]
Another guy did an item descs PR that conflicts heavily with this. You may need to make another one?

**Pull request closed without merging**